website/sidebars.js

```diff
@@ -99,19 +98,16 @@
       label: 'Cluster Management',
       link: {
         type: 'doc',
-        id: 'cluster-management/intro',
+        id: 'cluster-management/cluster-management-intro',
       },
       items: [
-        'cluster-management/getting-started',
-        'cluster-management/cluster-api-providers',
-        'cluster-management/managing-existing-clusters',
-        'cluster-management/provider-identities',
-        'cluster-management/deleting-a-cluster',
+        'cluster-management/managing-clusters-without-capi',
+        'cluster-management/deploying-capa-eks',
         'cluster-management/profiles',
-        'cluster-management/add-applications',
-        'cluster-management/gitrepo-selection',
-        'cluster-management/disable-capi',
-      ],
+        'cluster-management/cluster-management-troubleshooting',
+          {type: 'doc',
```

Comment:
You probaly wanted this under a new subsection `Advanced cluster management` but it appears as another link under the `Cluster Management` section.